aws-deadline/deadline-cloud-for-maya
# fix: ensure defaultArnoldRenderOptions are loaded when using mtoa

### What was the problem/requirement? (What/Why)
The Maya Submitter intermittently throws an exception upon opening. This only happens when the mtoa plugin is loaded.

### What was the solution? (How)
When the submitter detects Arnold as the renderer, it will try to access Arnold properties, however we are never loading the Arnold options. 

Now before we try to access those properties we ensure the mtoa `createOptions` has been called.

### What is the impact of this change?
The Submitter no longer throws an exception when the mtoa plugin is loaded. 

### How was this change tested?
Installed this version of the plugin on my local Maya 2023 install, then:

- Open Maya
- Ensure mtoa.mll is loaded
- Open and save an empty Maya scene.
- Open the Deadline Cloud submitter

Previously these steps would trigger an exception.

### Did you run the "Job Bundle Output Tests"? If not, why not? If so, paste the test results here.

No, this change only effects plugin loading. 

### Was this change documented?

No, it's only fixing a regression.

### Is this a breaking change?

No.
